### PR TITLE
Add GitHub actions

### PR DESCRIPTION
**Pull request by nishant-nayak**

### CODE REVIEW

File: initpost.sh [hunk 1]
```diff
@@ -1,218 +0,0 @@
-#!/usr/bin/env bash
```
Comment (gaurang2001) [inline initpost.sh]: Pls don't delete this

File: gulpfile.js [hunk 2]
```diff
@@ -1,98 +0,0 @@
-var gulp        = require('gulp'),
```
Comment (gaurang2001) [inline gulpfile.js]: What is this used for? You sure it's absolutely useless?

File: initpost.sh (hunk 1 above)
Comment (nishant-nayak) [reply initpost.sh]: Oh do we actually not want these deleted?

File: gulpfile.js (hunk 2 above)
Comment (nishant-nayak) [reply gulpfile.js]: Yeah, pretty sure gulpfiles aren't being used in the build process

**Review by anuragkumar2121 — APPROVED** [review]

**Review by anuragkumar2121 — DISMISSED** [review]

**Review by gaurang2001 — APPROVED** [review]

**Pull request merged**